recipe/session/sessmodels/models.go

```diff
@@ -133,8 +137,9 @@ type JWTNormalisedConfig struct {
 }
 
 type VerifySessionOptions struct {
-	AntiCsrfCheck   *bool
-	SessionRequired *bool
+	AntiCsrfCheck                 *bool
+	SessionRequired               *bool
+	OverrideGlobalClaimValidators func(globalClaimValidators []claims.SessionClaimValidator, sessionContainer *SessionContainer, userContext supertokens.UserContext) []claims.SessionClaimValidator
```

Comment:
should this function not be allowed to return an error as well?

Reply:
Done